doc/md/getting-started.md

```diff
@@ -103,11 +108,20 @@ ent
 ├── config.go
 ├── context.go
 ├── ent.go
+├── enttest
+│   └── enttest.go
+├── generate.go
+├── hook
+│   └── hook.go
 ├── migrate
 │   ├── migrate.go
 │   └── schema.go
+├── mutation.go
 ├── predicate
 │   └── predicate.go
+├── runtime
+│   └── runtime.go
+├── runtime.go
```

Comment:
Some lines are not needed here imo (at least in this stage). Consider changing it to:

```suggestion
├── generate.go
├── mutation.go
... truncated
├── schema
│   └── user.go
```